Gruntfile.js

```diff
@@ -447,43 +462,69 @@ module.exports = function( grunt ) {
 					packageManager: "npm"
 				}
 			}
+		},
+		"start-selenium-server": {
+			dev: {
+				options: {
+					downloadUrl: "https://selenium-release.storage.googleapis.com/2.45/" +
+						"selenium-server-standalone-2.45.0.jar",
+					downloadLocation: "node_modules/grunt-selenium-server/",
+					serverOptions: serverOptions,
+					systemProperties: {}
+				}
+			}
+		},
+		"stop-selenium-server": {
+			dev: {}
```

Comment:
@rxaviers What your talking about in your comment below is rather different. This is for testing via webdriver locally it very much aids in contributors being able to do with with out the steps i list above browserstack is great but most contributors wont have access to this.

Intern supports 3 ways of running tests.

Via Selenium with webdriver
Via Node with a runner
Via the browser directly via client.html

I think contributors should be able to easily do all three of these locally for proper testing this addresses # 1 what you ask for in the comment below addresses # 3